generators/server/templates/src/test/java/package/cucumber/CucumberTestContextConfiguration.java.ejs

```diff
@@ -18,13 +18,12 @@
 -%>
 package <%= packageName %>.cucumber;
 
-import <%= packageName %>.<%= mainClass %>;
+import <%= packageName %>.IntegrationTest;
```

Comment:
Are you sure it should reference "<%= packageName %>.IntegrationTest"?

Reply:
Yes, otherwise it fails. Also it promotes reuse: after all Cucumber tests are no more than integration tests